linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading_delete/ansible/shared.yml

```diff
@@ -4,7 +4,7 @@
 # disruption = low
 # strategy = configure
 
-{{% if "ol" in product or 'rhel' in product %}}
+{{% if "ol" in product or 'rhel' in product or 'ubuntu' in product %}}
```

Comment:
you should add `multi_platform_ubuntu` to this ansible platforms

Reply:
Thank you for the feedback. This was originally added in this commit in the original PR but the commit was modifying too many rules. I'll add this in when I get the chance

https://github.com/ComplianceAsCode/content/commit/e7d99c03246ebc5eacba43156987e3548a6ea73e

Reply:
Added!